packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -156,25 +156,30 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
     final List<Widget> result = <Widget>[];
 
     for (int index = 0; index < items.length; index += 1) {
+      final bool active = index == currentIndex;
       result.add(
         _wrapActiveItem(
           new Expanded(
-            child: new GestureDetector(
-              behavior: HitTestBehavior.opaque,
-              onTap: onTap == null ? null : () { onTap(index); },
-              child: new Padding(
-                padding: const EdgeInsets.only(bottom: 4.0),
-                child: new Column(
-                  mainAxisAlignment: MainAxisAlignment.end,
-                  children: <Widget> [
-                    new Expanded(child: new Center(child: items[index].icon)),
-                    items[index].title,
-                  ],
+            child: new Semantics(
+              selected: active,
+              hint: 'tab, ${index + 1} of ${items.length}',
```

Comment:
What about localization? We have a localized string for this in the MaterialLocalizations, not sure if that one can be reused here?

Reply:
We can't import package material. We'd have to solve #13452 first

Reply:
Understood. Can you add a TODO here to remind us that this needs localization?

Reply:
sg, done